transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Controllers/UserRequestControllerImp.java

```diff
@@ -21,15 +22,38 @@ public class UserRequestControllerImp implements UserRequestController {
 
     @Override
     @PostMapping("/storeAPI")
-    public void storeAPIInfo(@RequestBody API api) {
-        transcriptService.storeAPIInfo(api);
+    public void storeAPIInfo(@RequestBody UserAPI api) {
+        //transcriptService.storeAPIInfo(api);
+        transcriptService.storeAPIInfoProperties(api);
     }
 
     @Override
     @GetMapping("/cleanTranscript") // this will return a list
     public ArrayList<ArrayList<ArrayList<String>>>  getCleanedTranscript() throws IOException {
         return TranscriptService.getJSONContent();
     }
+
+    @GetMapping("/cleanTranscript/level/1") // this will return a list
```

Comment:
What is this route supposed to do? Try to document it more so the purpose is clearly understood. For example, I am not sure what is meant by "level 1".